### PR TITLE
BaSP-MR-150: Fix buttons and forms

### PR DESCRIPTION
**Pull request by SantiH97**
I fixed the buttons and forms of the admin view

### CODE REVIEW

**Comment by SantiH97** [comment]
[FIX IN ADMIN SCREEN](https://trello.com/c/04BtMSc6/150-fix-in-admin-screen)

File: src/Components/Members/Edit/EditMembers.jsx [hunk 1]
```diff
@@ -114,7 +114,8 @@ const MembersEdit = () => {
 
   useEffect(() => {
     dispatch(getMembers());
-    const memberToUpdate = data.find((member) => member._id === user._id);
+    const memberToUpdate = data.find((member) => member._id === id);
+    console.log(user);
```
Comment (DelfiLT) [inline src/Components/Members/Edit/EditMembers.jsx:118]: Remove console.log

**Review by DelfiLT — COMMENTED** [review]
Everything working perfectly! Only a few comments about the style of classes form:
The title should be in the middle and the buttons shouldn't be so far apart.

And also, in this PR you could add the classes validation to create a class with the activities that the trainer has to use the trainer's recent update!

File: src/Components/Classes/Form/index.jsx [hunk 2]
```diff
@@ -264,36 +264,30 @@ const Form = () => {
...
-                placeholder={id ? singleClass.trainerId?.firstName : 'Trainer'}
-                options={updatedTrainers}
-                nameValue={'trainerId'}
-                error={errors.trainerId?.message}
-                label={'Trainer'}
-              />
-            </div>
-          </div>
-          <div className={styles.selectContainer}>
+            <Select
```
Comment (DelfiLT) [inline src/Components/Classes/Form/index.jsx:267]: Add divs to every select!

**Review by DelfiLT — APPROVED** [review]

**Review by ukiweskamp — APPROVED** [review]

**Review by Dannylez — APPROVED** [review]

**Review by Eliezer-Alberto — APPROVED** [review]

**Pull request merged**